<ui version="4.0" >
 <class>ConverterGui</class>
 <widget class="QDialog" name="ConverterGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Converter for Opus Project Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QStackedWidget" name="stack_steps" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_error" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="p0_info_icon" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="pixmap" >
             <pixmap resource="resources.qrc" >:/images/images/messagebox_info.png</pixmap>
            </property>
            <property name="margin" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="p0_warning_icon" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="pixmap" >
             <pixmap resource="resources.qrc" >:/images/images/messagebox_warning.png</pixmap>
            </property>
            <property name="margin" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="p0_text" >
            <property name="text" >
             <string>error message</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLineEdit" name="le_filename" >
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="frame" >
             <bool>true</bool>
            </property>
            <property name="readOnly" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="pb_get_filename" >
            <property name="toolTip" >
             <string>Select file to convert</string>
            </property>
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_next" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Inspect</string>
           </property>
           <property name="icon" >
            <iconset resource="resources.qrc" >
             <normaloff>:/images/images/forward.png</normaloff>:/images/images/forward.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut" >
            <string>Ctrl+K</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QFrame" name="frame_page2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <item>
             <widget class="QLabel" name="p1_ok_icon" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/images/images/apply.png</pixmap>
              </property>
              <property name="margin" >
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="p1_info_icon" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/images/images/messagebox_info.png</pixmap>
              </property>
              <property name="margin" >
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="p1_warning_icon" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/images/images/messagebox_warning.png</pixmap>
              </property>
              <property name="margin" >
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="p1_text" >
              <property name="text" >
               <string>information</string>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_show_warnings" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>warnings</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_2" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6" >
             <item>
              <widget class="QLineEdit" name="le_filename_out" >
               <property name="font" >
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="frame" >
                <bool>true</bool>
               </property>
               <property name="readOnly" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="pb_get_filename_out" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_log" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="txt_log" >
            <property name="lineWrapMode" >
             <enum>QTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QPushButton" name="pb_back" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Inspect another file</string>
           </property>
           <property name="icon" >
            <iconset resource="resources.qrc" >
             <normaloff>:/images/images/back.png</normaloff>:/images/images/back.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut" >
            <string>Ctrl+K</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_log" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Log</string>
           </property>
           <property name="icon" >
            <iconset resource="resources.qrc" >
             <normaloff>:/images/images/messagebox_info.png</normaloff>:/images/images/messagebox_info.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut" >
            <string>Ctrl+K</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="flat" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_save" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Save</string>
           </property>
           <property name="icon" >
            <iconset resource="resources.qrc" >
             <normaloff>:/images/images/filesave.png</normaloff>:/images/images/filesave.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut" >
            <string>Ctrl+K</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
